CHANGELOG.rst

```diff
@@ -2,6 +2,14 @@
 Changelog
 =========
 
+0.20.0 (????)
+=============
+
+New Features:
+
+* Add optional support for ``django setup configuration``
```

Comment:
```suggestion
* Add optional support for `django-setup-configuration`_
```

& then add a link to the PyPI page of the package

Reply:
Done